readme.md

```diff
@@ -1424,13 +1424,20 @@ your `vite.config.js`.
 
 #### WMR
 
-~~[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
-[pushing them to `config.plugins`](https://github.com/preactjs/wmr/tree/main/packages/wmr#configuration-and-plugins)
-in `wmr.config.js`.~~
+[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
+[adding them to `plugins`](https://wmr.dev/docs/plugins)
+in `wmr.config.mjs`.
```

Comment:
Changed this to `.mjs` as [that's what the default WMR template ships with](https://github.com/preactjs/wmr/blob/main/packages/create-wmr/tpl/wmr.config.mjs). `.js` is fully supported, as is `.ts`, so not a huge deal either way.